examples/CSR/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "qrscanner_csr"
+version = "0.1.0"
+edition = "2021"
+authors = ["Dominik Wilkowski <Hi@Dominik-Wilkowski.com>"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+leptos = { version = "0.6", features = ["csr", "nightly"] }
+leptos_meta = { version = "0.6", features = ["csr", "nightly"] }
+leptos_router = { version = "0.6", features = ["csr", "nightly"] }
```

Comment:
`nightly` not necessary here, will fix it in a follow-up since you PRed your master and (if I even can, idk if you unchecked the "maintainers can push to branch" option) don't want to push on there.

Reply:
Good call but this is just the example so I wouldn't worry too much.

Reply:
I just couldn't run it because the nix dev shell I (ab-)use to build this doesn't have nightly :laughing: I should just build a dedicated nix shell for this project that optionally offers nightly for the SSR example. Normally I like not being on nightly by default, it keeps me from using features I shouldn't :sweat_smile:

Reply:
That's understandable. I never use nightly for anything other than leptos.